core/templates/domain/objects/FractionObjectFactory.ts

```diff
@@ -94,6 +94,16 @@ export class Fraction {
   isImproperFraction(): boolean {
     return this.denominator <= this.numerator;
   }
+
+  isEqualTo(fractionToCompare: Fraction): boolean {
```

Comment:
Do we expect this method to return True for the comparison of 2/4 and 1/2?

Reply:
Yes, changed it to do the same.